### DISCUSSION TITLE
Project BETA: Archive all column impossible

### DISCUSSION THREAD

**Discussion by xavierraffin** [community/community#5895] [Projects and Issues] [answered]
In the old project system I could archive all card of a column at once:

![Screen Shot 2021-09-29 at 6 24 41 PM](https://user-images.githubusercontent.com/1170030/135370356-dc3a99bd-dbfd-46d4-8d86-9eea4196fcf0.png)

It seems to be impossible now.

**Comment by xavierraffin** [comment]
Duplicate of https://github.com/github/feedback/discussions/5846 ?

**Comment by azenMatt** [comment, marked as answer]
Hi @xavierraffin - we will likely add a more UI friendly way to do this in the future, for the moment the quickest way is to hit `cmd+a` for select all (which on the first press will select all in a column) and then hit `e`

**Comment by ericboehs** [comment]
It looks like this has been added back to the board view. It doesn't work if your card is in the "No status" column though.

Also, there doesn't appear to be a way to archive all on a table view. Would be great if you could archive all in a grouped header (mousing over the header could add the down caret for Archive/delete like it does for each issue).
